tinydrive/configuration.md

```diff
@@ -83,6 +83,26 @@ tinymce.init({
 
 ## Configuring the Tiny Drive UI
 
+### tinydrive_skin
+
+This options sets the skin applied to {{site.cloudfilemanager}}. The default skin included with {{site.cloudfilemanager}} is named "oxide".
+
+**Type:** `String`
+
+**Default Value:** `oxide`
+
+**Possible Values:** `oxide`, `oxide-dark`
```

Comment:
```suggestion
**Default Value:** `'oxide'`

**Possible Values:** `'oxide'`, `'oxide-dark'`
```

As this is a string, we normally add the quotes so it can be copied/pasted easier.